### PR TITLE
Add another intro section in Donation page

### PR DESCRIPTION
**Pull request by sarunast**
Added centered intro block after the introduction.

screen shot 2018-02-07 at 13 26 04

It would be best if @erikjalevik would look at this since I didn't have designs to do this.

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit 3f77a84d8f0f0765de24e210ca86b27422cf8d9d

https://deploy-preview-143--masifunde.netlify.com

File: pages/how-to-support/donate.js [hunk 1]
```diff
@@ -47,6 +50,10 @@ const Donate = ({
       <CenteredText source={introMarkdown} />
     </PageSection>
 
+    <FormContainer>
+      <Markdown source={intro2Markdown} />
+    </FormContainer>
```
Comment (torkjels) [inline pages/how-to-support/donate.js]: The contenst of this `FormContainer` is not a form at all. Should we consider renaming the `FormContainer`, or possibly going for a different solution here?

**Comment by torkjels** [comment]
Looks good! 👍 Should this style maybe also be applied to `<ol>`? So they don't have to remember they have to use a bullet list and not an ordered list.

**Comment by torkjels** [comment]
Hm yeah, good point. In this section they said they wanted just a list with check marks as bullet points, but who am I to say that they'd never want a numbered list? I'll keep it as is then, only styling the `<ul>`.

**Pull request merged**